dev-lang/ghc/ghc-9.0.2-r4.ebuild

```diff
@@ -597,6 +598,22 @@ src_prepare() {
 		eapply "${FILESDIR}"/${PN}-9.0.2-fptools.patch # clang-16 workaround
 		eapply "${FILESDIR}"/${PN}-9.0.2-sphinx-6.patch
 
+		# FIXME: A hack that allows dev-python/sphinx-7 to build the docs
+		#
+		# GHC has updated the bundled version here:
+		# <https://gitlab.haskell.org/ghc/ghc/-/commit/70526f5bd8886126f49833ef20604a2c6477780a>
+		# However, the patch is difficult to apply and our versions of GHC don't
+		# have the update, so we symlink to the system version instead.
+		if use doc; then
+			local rtd_theme_dir="$(dirname $(python -c "import sphinx_rtd_theme; print(sphinx_rtd_theme.__file__)"))"
+			local orig_rtd_theme_dir="${S}/docs/users_guide/rtd-theme"
+
+			ebegin "Replacing bundled rtd-theme with dev-python/sphinx-rtd-theme"
+			rm -r "${orig_rtd_theme_dir}" || die
+			ln -s "${rtd_theme_dir}" "${orig_rtd_theme_dir}" || die
```

Comment:
Note that because of the `|| die` here (which is fine/right), you can't ever get to the `eend` below with anything other than success.

I'd just do `einfo` instead of the ebegin/eend.

Reply:
That was my logic: might as well run `eend 0` since we know `$?` is going to be `0`. I just wanted the aesthetic of having an `[ ok ]` when it works, but `einfo` is fine too.

Reply:
Fixed in 18bd96a